NeoMathEngine/src/CPU/x86/CpuX86MathEngineVectorMath.cpp

```diff
@@ -37,24 +37,22 @@ void CCpuMathEngine::VectorCopy( const CIntHandle& firstHandle, const CConstIntH
 	dataCopy( GetRaw( firstHandle ), GetRaw( secondHandle ), vectorSize );
 }
 
-void CCpuMathEngine::vectorCopy( const CFloatHandle& firstHandle, const CConstFloatHandle& secondHandle, int vectorSize )
+void CCpuMathEngine::vectorCopy( float* firstHandle, const float* secondHandle, int vectorSize )
 {
-	dataCopy( GetRaw( firstHandle ), GetRaw( secondHandle ), vectorSize );
+	dataCopy( firstHandle, secondHandle, vectorSize );
 }
 
 void CCpuMathEngine::VectorFill( const CFloatHandle& result, float value, int vectorSize )
```

Comment:
There're lots of places inside of `CCpuMathEngine` methods, where `VectorFill` is called (or `VectorCopy` or any other function that has faster `float*` version).

I recommend you to replace all of them with these new versions (if possible)

E.g. `CCpuMathEngine::SetVectorToMatrixRows` or `CCpuMathEngine::SumMatrixRows`